### DISCUSSION TITLE
Using Zustand for Global State and Jotai for Local State

### DISCUSSION THREAD

**Discussion by dzmitryFesik** [pmndrs/jotai#2686] [General] [answered]
I'm considering using Zustand for global state management and Jotai for local state (React.Context + React.State replacement)

- What are the pros and cons?
- Any potential issues or performance concerns?

Looking forward to your insights!

**Comment by dai-shi** [comment, marked as answer]
I remember someone suggested this pattern.
I wouldn't personally do it, but actually it's reasonable.
Yeah, I'm also interested in how people think about it.

There are no technical concerns per se.
